Dockerfile

```diff
@@ -0,0 +1,27 @@
+# syntax=docker/dockerfile:1
+
+## Build
+FROM golang:1.18-buster AS build
+
+WORKDIR /usr/local/go/src/github.com/spyderbat/event-forwarder
+
+COPY ./ ./
+RUN go mod tidy
+RUN CGO_ENABLED=1 go build -trimpath -ldflags "-s -w" -o /event-forwarder ./spyderbat-event-forwarder
```

Comment:
I like this,  this is a simple one so this makes sense.    I've been putting this together for other builds: https://github.com/spyderbat/agentbuild